### PR TITLE
Add ruby 3.1 to build matrix

### PR DESCRIPTION
**Pull request by sue445**

### CODE REVIEW

File: template/gem_name.gemspec.erb [hunk 1]
```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'simplecov', '~> 0.21.0'
 
-  spec.add_development_dependency 'rubocop', '~> 1.21.0'
```
Comment (AlexWayfer) [inline template/gem_name.gemspec.erb]: I recommend to separate this change to another PR.

Comment (sue445) [reply template/gem_name.gemspec.erb]: I've split this patch into #6

**Review by olleolleolle — APPROVED** [review]
Thanks a lot!
